Add meta descriptions (#4003)
wazuh/wazuh-documentation: source/deploying-with-ansible/index.rst

.. Copyright (C) 2021 Wazuh, Inc.

.. meta::

>   :description: Check out this guide to learn how to deploy Wazuh with Ansible, an open source platform designed for automating tasks.

Reviewer: ```suggestion
  :description: Check out the Wazuh documentation to learn how to deploy Wazuh with Ansible, an open source platform designed for automating tasks.
```
